internal/policy/policy.go

```diff
@@ -695,3 +694,17 @@ func (s *State) findDelegationEntry(roleName string) (tuf.Delegation, error) {
 		}
 	}
 }
+
+func ListRules(ctx context.Context, repo *git.Repository) ([]tuf.Delegation, error) {
+	state, err := LoadCurrentState(ctx, repo)
+	if err != nil {
+		return nil, err
+	}
+
+	targetsMetadata, err := state.GetTargetsMetadata(TargetsRoleName)
+	if err != nil {
+		return nil, err
+	}
+
+	return targetsMetadata.Delegations.Roles, nil
```

Comment:
let's skip displaying the allow rule

Reply:
I have not changed the ListRules function in policy, instead I just excluded showing it when it was being outputted.